.checkignore

```diff
@@ -0,0 +1 @@
+test/*
```

Comment:
Is there any way to disable warnings for Test classes? QC complains about comments for each test.